lib/workers/repository/process/fetch.ts

```diff
@@ -50,10 +50,12 @@ async function fetchManagerPackagerFileUpdates(
   const { packageFile } = pFile;
   const packageFileConfig = mergeChildConfig(managerConfig, pFile);
   const { manager } = packageFileConfig;
-  const queue = pFile.deps.map(
-    (dep) => (): Promise<PackageDependency> =>
-      fetchDepUpdates(packageFileConfig, dep)
-  );
+  const queue = pFile.deps
+    .filter((dep) => dep.depName?.length)
```

Comment:
I'm not sure if this is the right place to fix the issue 🤔

Reply:
Oh where do you think is the right place? 
I've debugged and saw this is the where the value is coming from (a few calls before getPkgReleases where the error is being logged)

Reply:
Ideally we could check extracted deps centrally and warn plus add a skip reason if they are missing a valid depName.

Reply:
@rarkins Do you think we should just check if the depName is empty and add a skip reason in fetchDepUpdates before line 18 where we check if there is a skip reason for the dep?